### PR TITLE
Add PLAYER_VS_PLAYER functionality to SDK and Controller

### LINKED ISSUE #17: SDK: Player vs Player

### LINKED ISSUE #18: Controller: Player vs Player

### PR DESCRIPTION
**Pull request by ballyalley-o**
This pull request adds the functionality to retrieve player vs player data to the SDK and Controller modules. It includes the following changes:

- Added the PLAYER_VS_PLAYER directory path to the SDK_DIR configuration object.

- Added the PLAYER_VS_PLAYER route to the SDK module.

- Updated the Franchise ownership field in the Player Module to be an array.

- Updated the properties of the Team model.

- Added the PLAYER_VS_PLAYER route to the SDK module.

- Updated the Team model properties.

- Created player base data in the SDKSetController.

- Created franchise history records based on the NBA API data in the SDKSetController.

- Added the getPlayerVsPlayer method to the SDKController to retrieve player vs player data.

- Added the getGameFinder method to the SDKController to retrieve game data.

These changes enhance the functionality of the application by providing a convenient way to access player vs player data and game data.

### CODE REVIEW

File: sdk/sdk-server.py [hunk 1]
```diff
@@ -80,6 +80,42 @@ def get_player_career(player_id):
...
+    Parameters:
+    player_id (int): The ID of the first player.
+    vs_player_id (int): The ID of the second player.
+
+    Returns:
+    dict: A dictionary containing the head-to-head statistics between the two players.
+
+    """
+
+    season = request.args.get('season', '2023-24')
```
Comment (in-a-iii) [inline sdk/sdk-server.py:98]: add all the params from sdk… make it available to the other end.

**Review by in-a-iii — CHANGES_REQUESTED** [review]

File: sdk/sdk-server.py (hunk 1 above)
Comment (ballyalley-o) [reply sdk/sdk-server.py:98]: will do.

Comment (ballyalley-o) [reply sdk/sdk-server.py:98]: done. please review.

**Review by in-a-iii — APPROVED** [review]
great work. good to go.

**Pull request merged**